packages/office-addin-sso/src/ssoDataSettings.ts

```diff
@@ -49,11 +53,12 @@ export function getSecretFromCredentialStore(ssoAppName: string, isTest: boolean
                 const getSecretFromMacStoreCommand = `${isTest ? "" : "sudo"} security find-generic-password -a ${os.userInfo().username} -s ${ssoAppName} -w`;
                 return execSync(getSecretFromMacStoreCommand, { stdio: "pipe" }).toString();;
             default:
+                usageDataHelper.sendUsageDataException('getSecretFromCredentialStore', `unsupported platform ${process.platform}`);
                 throw new Error(`Platform not supported: ${process.platform}`);
         }
-
     } catch (err) {
-        return '';
```

Comment:
this code shouldn't have been changed - reverting back locally and will recommit